src/alchemlyb/preprocessing/subsampling.py

```diff
@@ -170,11 +170,12 @@ def statistical_inefficiency(df, series=None, lower=None, upper=None, step=None,
                            "values are sorted by time, increasing.")
 
     if series is not None:
-        series = slicing(series, lower=lower, upper=upper, step=step)
-
+    
         if (len(series) != len(df) or
             not all(series.reset_index()['time'] == df.reset_index()['time'])):
             raise ValueError("series and data must be sampled at the same times")
```

Comment:
Do we test this ValueError anywhere?

Reply:
According to https://app.codecov.io/gh/alchemistry/alchemlyb/compare/167/diff we do, but not sure where.